### ISSUE TITLE
OTR: children of GO:0048870 cell motility

### ISSUE THREAD

**Issue by paolaroncaglia** [geneontology/go-ontology#12722]
Wrapping up cilia-related tickets, this one is for future work:

From an old email from Chris to the ontology group:

“I think it's worth looking at the mix of asserted children here and trying to approach this differently

is_a GO:0044763 ! single-organism cellular process
is_a GO:0006928 ! cellular component movement
is_a GO:0048870 ! cell motility
is_a GO:0001539 ! cilium or flagellum-dependent cell motility
is_a GO:0003411 ! cell motility involved in camera-type eye morphogenesis
is_a GO:0016477 ! cell migration
is_a GO:0021805 ! cell movement involved in somal translocation
is_a GO:0021814 ! cell motility involved in cerebral cortex radial glia guided migration
is_a GO:0030317 ! sperm motility
is_a GO:0043107 ! type IV pilus-dependent motility
is_a GO:0070358 ! actin polymerization-dependent cell motility
is_a GO:0071975 ! cell swimming
is_a GO:0071976 ! cell gliding
is_a GO:0090247 ! cell motility involved in somitogenic axis elongation
is_a GO:0097230 ! cell motility in response to potassium ion
is_a GO:0097231 ! cell motility in response to calcium ion

What are the different mechanisms of motility? Are they disjoint? Can something be 'cell swimming' and cilium-directed and migration at the same time?

It looks like the differentia are a mix of mechanism (e.g. cilium, pilus), context (involved in, response to) or a descriptive measure of the motion or immediate environment (swimming, gliding). Or migration which seems insufficiently differentiated.

On closer examination of the children, the last category seems to disappear and that these are actually cryptic mechanism descriptions too: gliding is achieved through secretion of adhesion molecules and 'swimming' is either the cytoskeleton contracting the cell or GO:0071977 which seems insufficiently differentiated from its parent

I'd prefer to ground things more mechanistically or contextually.

Once this is done it gives us a better way to answer questions about where 'sperm motility' can go. Are there any other possible mechanisms other than cilium? If not, then a hidden GCI is justified. Does this hold for any cell with an X as part, where X is a possible mechanism of motility?”

**Comment by paolaroncaglia** [comment]
Moving to https://github.com/geneontology/go-ontology/issues/12783.
